http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -153,6 +154,16 @@ if [[ "$RUNTIME_OPERATION" == 'creating' ]]; then
 
     mkdir -p /work
     mkdir -p /certs
+    # Mount persisent disk
+    export DISK_DEVICE_ID=$(lsblk -o name,serial | grep 'user-disk' | awk '{print $1}')
+    # Only format disk is it hasn't already been formatted
+    if [ "$IS_GCE_FORMATTED" == "false" ] ; then
+      mkfs.ext4 -m 0 -E lazy_itable_init=0,lazy_journal_init=0,discard /dev/${DISK_DEVICE_ID}
+    fi
+    mount -o discard,defaults /dev/${DISK_DEVICE_ID} /work
+    # Ensure peristent disk re-mounts if runtime stops and restarts
+    cp /etc/fstab /etc/fstab.backup
+    echo UUID=`blkid -s UUID -o value /dev/${DISK_DEVICE_ID}` /work ext4 discard,defaults,nofail 0 2 | tee -a /etc/fstab
```

Comment:
Cool - so this means we don't need any changes to `startup.sh`?

Reply:
yep, i tested and validated that a PD gets mounted after restarting an instance